src/App.js

```diff
@@ -7,64 +7,47 @@ import { getTodos, getUsers } from './api/api';
 
 class App extends React.Component {
   state = {
-    todosNew: [],
     todos: [],
     users: [],
+    user: {},
     selectedUserId: 0,
     input: '',
     select: '',
```

Comment:
store this values in the TodosList components state, entire app shouldn't know about filtering variables